orchestration/templates/ingest-clinvar-archive.yaml

```diff
@@ -162,8 +148,23 @@ spec:
                   value: '{{ .Values.gcs.bucketName }}'
                 - name: gcs-file-path
                   value: '{{ "{{inputs.parameters.gcs-prefix}}" }}/raw/{{ $localXmlName }}'
-          {{- $extractionPvc := "{{tasks.generate-extraction-volume.outputs.parameters.pvc-name}}" }}
-          # 7b) Run Dataflow on uploaded JSON.
+          # 7) Stage a release_history object in GCS
+          - name: stage-release-history
+            dependencies: [extract-release-date, ingest-raw-xml]
+            templateRef:
+              name: create-json-object
+              template: main
+            # WHEN the file does not already exist in the repo, THEN we can expect a drs-id for a file
+            when: '{{ "{{tasks.ingest-raw-xml.outputs.parameters.file-exists}} == false" }}'
```

Comment:
Can we test one re-run to make sure this `when` does what we expect? Or is that effectively blocked until we fix the `when` logic in the ingest-file template?

Reply:
I can delete the ingested xml blob and run it again

Reply:
wait jk you're right